test/deals/post-deals.api-test.ts

```diff
@@ -0,0 +1,78 @@
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES } from '@ukef-test/support/environment-variables';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+describe('POST /deals', () => {
+  const valueGenerator = new RandomValueGenerator();
```

Comment:
Can we please add more test assertions where:

* dealId is not a valid value.
* Payload has missing attributes.
* Payload has invalid formatted attributes.